[Fix] Left panel should display no parser message. Fix #1740

ewoken opened the pull request:
Should we remove "hr" from display ? 

PS: it is my first contribution, be indulgent. ;-)

systemcatch commented:
Hi @ewoken, thanks for the contribution. Can you build the map locally (https://github.com/tmrowco/electricitymap-contrib#running-locally) so we can see how this looks in a screenshot.

systemcatch commented:
Fantastic, that looks to be working great. I'm not sure what you mean by margin above message & timeslider though. Maybe try making the changes on your local machine as a demonstration.

corradio commented:
@ovbm can you review?

ovbm commented on web/src/components/countrytable.js:
```diff
@@ -625,7 +625,13 @@ CountryTable.prototype.electricityMixMode = function(arg) {
+    .country-show-emissions-wrap,
+    .country-panel-wrap .bysource,
+    .country-table-container,
+    .country-history
```
```suggestion
    .country-history,
    .zone-time-slider, 
    hr
```

ewoken commented:
<img width="200" alt="capture d ecran 2019-01-24 a 11 37 14" src="https://user-images.githubusercontent.com/8919530/51672724-78ae7780-1fcc-11e9-977d-074dea788b14.png"><img width="200" alt="screenshot_2019-01-24 emissions co2 de la consommation electrique en temps reel" src="https://user-images.githubusercontent.com/8919530/51672746-82d07600-1fcc-11e9-8ceb-13dce05e4ebc.png">
As you suggested me, I've added a margin-bottom in the desktop case. I also added a margin-top for mobile in order to separate social-buttons from text.
